create-plugin: Add provisioning scaffold (#529)
grafana/plugin-tools: docusaurus/docs/publish-a-plugin/publish-or-update-a-plugin.md

### How long does it take to review a submission?

> - We're not able to give an estimate at this time, though we're constantly working on improving the time it takes to review a plugin. We have provided an optional [provisioning](./provide-provisioning.md) process to speed up reviews.

Reviewer: ```suggestion
- We're not able to give an estimate at this time, though we're constantly working on improving the time it takes to review a plugin. Providing a [provisioned](./provide-provisioning.md) test environment can drastically speed up your review.
```

Reviewer 2: done.
